test/include/nntrainer_test_util.h

```diff
@@ -122,6 +127,21 @@ typedef enum {
 class nntrainerIniTest
   : public ::testing::TestWithParam<
       std::tuple<const char *, const std::vector<IniSection>, int>> {
+public:
+  static void save_ini(const char *filename, std::vector<IniSection> sections,
```

Comment:
FYI, There was a major refactoring on ini helper classes.

https://github.com/nnstreamer/nntrainer/pull/661/commits/4f17acd6e406421eb69e2ee4ccd9fa42e6b44c7c

1. `nntrainerIniTest` has moved to `nntrainer_modelfile.cpp` since it is a tailored class for ini test
2. Instead, there is `IniTestWrapper` for general usage of `Ini` handling. So, you can create an instance for each ini file. As it is merged, IMHO, it removes the need of this function and `getIni` function. (filemode argument can be added to IniTestWrapper for the `getIni`)

I should've reviewed this PR earlier and told you beforehand. Sorry about that :(

Reply:
its alright, I will update with rebase.